nibabel/freesurfer/io.py

```diff
@@ -160,6 +160,29 @@ def read_morph_data(filepath):
     return curv
 
 
+def write_morph_data(filepath, values):
+    """Write out a Freesurfer morphometry data file.
+
+    See:
+    http://www.grahamwideman.com/gw/brain/fs/surfacefileformats.htm#CurvNew
+
+    Parameters
+    ----------
+    filepath : str
```

Comment:
Not that I know of. Would the convention for that would be the following?

    file_like : file-like